codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
```

Comment:
I feel so stupid for not realizing I could do this..  I spent so long trying to see how to exclude from `go test`.

Reply:
I also spent a few hours trying also to change the test scripts in `makefile` to exclude those files :sweat_smile: 
then I found it in their [documentation](https://docs.codecov.com/docs/ignoring-paths)

Reply:
Is there a way to see the new coverage stats for this PR ? I looked into the coverage CI logs  and followed the link of the success status : https://app.codecov.io/github/armadaproject/armada/commit/216017c33006d880ed658cd14299703eeb3f0673
but it produces and error.

Reply:
I get the same error when trying to view that results link as well. I'll try again in an hour, and see if it works.

Reply:
There's a JS error on that page - hopefully they are fixing it right now.

Reply:
I just checked another PR (one for Lookout V2, by Carlo Camurri) in this repo, and the CodeCov results link is broken for that one as well, so I think this is very much a CodeCov issue for multiple PRs/repos.  I also found another armada PR where the upload of the CodeCov results returns a "Server 500" error from CodeCov, so they are having multiple problems.

Reply:
https://app.codecov.io/github/armadaproject/armada/tree/exclude-generated-code-cov

Yea this tool is not the best..  You get what you pay for!

Reply:
Looks like this PR did not work though.

Reply:
<img width="1144" alt="Screenshot 2023-03-01 at 3 08 29 PM" src="https://user-images.githubusercontent.com/3780425/222254340-b898ff4b-5120-4017-9a75-915dfd41190f.png">

Reply:
hmmm, I will try something. may be I will exclude a specific file or a directory without regex to see the if this `codecov.yml` is working at all.

Reply:
<img width="1293" alt="Screen Shot 2023-03-01 at 1 14 41 PM" src="https://user-images.githubusercontent.com/3155623/222255455-d33608dc-730c-4928-b923-69bf760e94f7.png">

Does this need to be merged in the `armada` repo, for it to actually take effect?

Reply:
I have no clue tbh. but I checked this [PR's](https://github.com/armadaproject/armada/pull/2169) branch [here](https://app.codecov.io/github/armadaproject/armada/tree/improve-internal-test-cov) and the effect appears without merging it (from 46.71% to 46.71% )

Reply:
I just excluded the `cmd` directory and  it worked 
![Screenshot from 2023-03-01 22-41-50](https://user-images.githubusercontent.com/39927413/222260449-b289886c-5eac-49e8-b926-8f956b817ec2.png)

So I believe the problem was with the regex